src/tests/ftest/get_remote_files.sh

```diff
@@ -0,0 +1,237 @@
+#
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the following disclaimer in the
+#    documentation and/or materials provided with the distribution.
+#
+# 3. In addition, redistributions of modified forms of the source or binary
+#    code must carry prominent notices stating that the original code was
+#    changed and the date of the change.
+#
+#  4. All publications or advertising materials mentioning features or use of
+#     this software are asked, but not required, to acknowledge that it was
+#     developed by Intel Corporation and credit the contributors.
+#
+# 5. Neither the name of Intel Corporation, nor the name of any Contributor
+#    may be used to endorse or promote products derived from this software
+#    without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+# THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#######################################
+# Print usage to user.
+#######################################
+usage() {
+    echo -n "get_remote_files.sh [OPTION]...
+
+This script has 3 modes that can be executed independently if needed.
+
+ 1. Display big files specified by threshold value i.e. -t 5mb. This option
+    will list files within provided local path with -d option and will prepend
+    each file with a \"Y\" if file exceeds threshold value and with \"N\" if
+    it does not.
+
+ 2. Compress files specified in provided local path with -d option that exceed
+    1M in size. This script will check if current local system is a VM host, if
+    so, compression will not be executed.
+
+ 3. Archive files specified in provided local path with -d option to provided
+    remote path destination.
+
+ Options:
+      -a        Archive files
+      -c        Compress files
+      -d        Local path to files for achiving/compressing/size checking
+      -r        Remote path to scp files to i.e. server-A:/path/to/file
+      -t        Threshold value to determine classification of big logs
+      -v        Display commands being executed
+      -h        Display this help and exit
+"
+}
+
+#######################################
+# Print set GLOBAL variables.
+#######################################
+display_set_vars() {
+    echo "Set Variables:
```

Comment:
Maybe only print this if "-v" is specified

Reply:
Done.